draft.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
```

Comment:
Are we going to use this file in the code?
It looks to me like a test file.

Reply:
Oh, you are correct, that is a test file. I believe it was uploaded either as a convenience or by mistake really... it will be removed in the next commit.